addons/website_forum/models/forum_post.py

```diff
@@ -336,7 +336,7 @@ def create(self, vals_list):
             # add karma for posting new questions
             if not post.parent_id and post.state == 'active':
                 post.create_uid.sudo()._add_karma(post.forum_id.karma_gen_question_new, post, _('Ask a new question'))
-        posts.post_notification()
+        posts.sudo().post_notification()
```

Comment:
Can't we take the opportunity to use a better name, like ``_notify_state_update`` ?